content/F2015/F15-event-bigcsters-balanced-life.md

```diff
@@ -0,0 +1,24 @@
+title: Big CSters Discussion Circle: Balancing Lifestyle
+Date: 2015-10-15
+Category: Events
+Tags: social, big csters
+Slug: big-csters-balanced-life-s16
+Author: Bhargavi Dameracharla
+Summary: Meet fellow CSters and discuss strategies to balance a busy life.
+
+Some of us have had times when we think one aspect of our life is more
+important than the other, such as studying over hanging out with friends
+or being with family. However, we should be aware that everything plays a
+role for our well-being. We will be sharing some tips and strategies to
+balance your super-busy student life. Come and meet your fellow CSters!
+
+Like previous events, dinner will be provided!
+
+Note: This event is [women-only](http://wics.uwaterloo.ca/faq).
```

Comment:
It's better practice to make these links relative, like this:

`[women-only]({filename}/pages/faq.md)